Wikipedia/Code/OnThisDayExploreCollectionViewCell.swift

```diff
@@ -2,25 +2,38 @@ import UIKit
 
 @objc(WMFOnThisDayExploreCollectionViewCell)
 class OnThisDayExploreCollectionViewCell: OnThisDayCollectionViewCell {
-
-    private lazy var whiteTopAndBottomGradientLayer: CAGradientLayer = {
-        let layer = CAGradientLayer()
-        let white = UIColor.white.cgColor
-        let clear = UIColor(white: 1, alpha: 0).cgColor // https://stackoverflow.com/a/24895385
-        layer.colors = [white, clear, clear, white, white]
-        layer.locations = [0.0, 0.07, 0.77, 0.97, 1.0]
-        layer.startPoint = CGPoint(x: 0.5, y: 0.0)
-        layer.endPoint = CGPoint(x: 0.5, y: 1.0)
-        return layer
-    }()
-
+    fileprivate var topGradientView: WMFGradientView = WMFGradientView()
```

Comment:
hmm... the single gradient layer seemed simpler/more concise. if we need to adjust heights why not just adjust the `layer.locations` stops above?